tests/scenario/test_grafana_cloud_integrator_lib.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from charms.grafana_cloud_integrator.v0.cloud_config_requirer import (
+    CloudConfigAvailableEvent,
+    CloudConfigRevokedEvent,
+    GrafanaCloudConfigRequirer,
+)
+from ops import CharmBase, Framework
+from scenario import Context, Relation, State
+
+
+class MyCharm(CharmBase):
+    def __init__(self, framework: Framework):
+        super().__init__(framework)
+        self.cloud = GrafanaCloudConfigRequirer(self)
+
+
+@pytest.fixture()
+def context():
```

Comment:
Depending on how we end up organizing tests, I wonder if `context` should be a `classmethod` (factory) of `MyCharm`.

Reply:
IMHO we should stick to the standard pattern of using a fixture for the context, to be consistent with other scenario testing suites.

Reply:
Personally I prefer fixtures.  The parameters to `Context()` could in theory vary between cases.  The fixture is essentially the factory method here.

Having said that, functionally I think both approaches are the same here.  I vote to leave it as is and we can always change it as things expand if it feels appropriate.

Reply:
Resolving given @Abuelodelanada's review, but if anyone disagrees we can open another issue and fix it